backend/src/main/kotlin/org/loculus/backend/service/submission/UpdateTrackerTable.kt

```diff
@@ -0,0 +1,8 @@
+import org.jetbrains.exposed.sql.Table
```

Comment:
This is missing it's package declaration. (Annoying, but we're still in the Java world)
```suggestion
package org.loculus.backend.service.submission

import org.jetbrains.exposed.sql.Table
```